fe/fe-core/src/main/java/com/starrocks/statistic/StatisticUtils.java

```diff
@@ -31,8 +31,12 @@ public static ConnectContext buildConnectContext() {
         // but QeProcessorImpl::reportExecStatus will check query id,
         // So we must disable report query status from BE to FE
         context.getSessionVariable().setReportSuccess(false);
-        // Always use 1 parallel to avoid affect normal query
-        context.getSessionVariable().setParallelExecInstanceNum(1);
+        int parallel = context.getSessionVariable().getStatisticCollectParallelism();
+        if (null != ConnectContext.get()) {
```

Comment:
When ConnectContext.get() will appear Null， and what different with context default session variable and ConnectContext.get().getSessionVariable()?

Reply:
AutoCollector may null, if you only use default session variable, you need set the variable by `set global`